bin/s3uploader

```diff
@@ -36,6 +36,10 @@ opt_parser = OptionParser.new do |opt|
     options[:s3_secret] = s3_secret
   end
 
+  opt.on("-g s","--grep *xml","set specific file or specific type files. Default: **/* recursive mode") do |filter|
```

Comment:
The short and long option must be defined as "-g PATTERN" and "--grep PATTERN", and then use PATTERN in the description. Eg.: "Filter full file paths using PATTERN as the regular expression"